### DISCUSSION TITLE
What are the requirements to become a maintainer ?

### DISCUSSION THREAD

**Discussion by SpidFightFR** [void-linux/void-packages#47741] [Q&A] [answered]
Hey everyone, everything is in the title. I was wondering out of curiosity.

Mesa is marked as orphaned since quite some time now, and even tho it isn't _bad_ itself, i was just wondering if there were any skills required to become a maintainer for mesa, for example.

I also ask this, seeing how people make patches that fix problems in updates, i quite struggle to understand how to do such things... And i'm afraid lacking such skills would block someone like me to become a maintainer.

Anyways, have a nice day !

**Comment by classabbyamp** [comment, marked as answer]
generally, see [CONTRIBUTING.md](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#adopting-a-template)

orphaned doesn't necessarily mean that there's *no* maintainer, more that there's no *specific* maintainer and the core team keeps watch over it, especially for core packages like mesa.

I would suggest trying to dig deeper and understand what errors you're running into instead of just stopping because of the errors before adopting a package like mesa

**Reply by SpidFightFR** [reply]
Hey, thanks for your reply !

Sorry to disturb, but would you have, by chance, any documentation to help me making patches like maintainers like you do ?
It's really impressive to see how you guys fix issues. I wish to learn more, to make myself more useful to Void-Linux !
